service/openapi.yaml

```diff
@@ -0,0 +1,285 @@
+  contact:
+    url: https://github.com/open-feature/protocol
+  license:
+    name: Apache 2.0
+security:
+  - ApiKeyAuth: []
+  - BearerAuth: []
+paths:
+  /ofrep/v1/evaluate/{key}:
+    post:
+      description: OFREP flag evaluation request
+      parameters:
+        - name: key
+          in: path
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/evaluationRequest'
+      responses:
+        '200':
+          description: OFREP successful evaluation response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/success'
+        '400':
+          description: OFREP error response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/failure'
+        '401':
+          description: Unauthorized - You need credentials to access the API
+        '403':
+          description: Forbidden - You are not authorized to access the API
+        '404':
+          description: Flag not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/failure'
+        '429':
+          description: Rate limit reached on the Flag Management System
+          headers:
+            Retry-Later:
+              description: Indicates when to retry the request
+              schema:
+                type: string
+                format: date-time
+                example: '2024-02-07T12:00:00Z'
+        '500':
+          description: Internal server error
+  /ofrep/v1/evaluate:
+    post:
+      description: OFREP bulk evaluation request
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/bulkEvaluationRequest'
+      responses:
+        '200':
+          description: OFREP successful evaluation response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/bulkEvaluationSuccess'
+        '400':
+          description: OFREP error response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/bulkEvaluationFailure'
+        '401':
+          description: Unauthorized - You need credentials to access the API
+        '403':
+          description: Forbidden - You are not authorized to access the API
+        '429':
+          description: Rate limit reached on the Flag Management System
+          headers:
+            Retry-Later:
+              description: Indicates when to retry the request
+              schema:
+                type: string
+                format: date-time
+                example: '2024-02-07T12:00:00Z'
+        '500':
+          description: Internal server error
+  /ofrep/v1/flag/changes:
```

Comment:
I think we should avoid spec'ing this endpoint, _at least_ for now. See my comment [here](https://github.com/open-feature/protocol/pull/2#discussion_r1496311806).